pkg/operator/csidriveroperator/driverstarter.go

```diff
@@ -56,8 +56,9 @@ type csiDriverControllerManager struct {
 	operatorConfig csioperatorclient.CSIOperatorConfig
 	// ControllerManager that installs the CSI driver operator and all its
 	// objects.
-	mgr     manager.ControllerManager
-	running bool
+	mgr                manager.ControllerManager
+	running            bool
+	ctrlRelatedObjects func() ([]configv1.ObjectReference, error)
```

Comment:
This looks worthy of an interface to me. I won't stick on the first one I see, but I'm likely to stick on it when we grow another.

Reply:
+1 👍🏽

Reply:
Please make an interface for this, the rest is lgtm